sdcm/remote.py

```diff
@@ -188,7 +188,9 @@ def __init__(self, hostname, user="root", port=22, connect_timeout=60, password=
                                  'UserKnownHostsFile': self.known_hosts_file,
                                  'ServerAliveInterval': 300,
                                  'StrictHostKeyChecking': 'no'})
-        self.connect_config = {'key_filename': os.path.expanduser(self.key_file)}
```

Comment:
please revert all changes here, we are only working with keys.
not with passwords